signer/src/transaction_coordinator.rs

```diff
@@ -351,12 +352,6 @@ where
         let transaction_package = pending_requests.construct_transactions()?;
 
         for mut transaction in transaction_package {
-            self.send_message(
```

Comment:
This is moved now because we need to store the size of the transaction after signing, which happens in `sign_and_broadcast()`.